app/src/main/java/fr/gaulupeau/apps/Poche/ui/ArticlesListActivity.java

```diff
@@ -447,6 +448,7 @@ private ArticlesListFragment getFragment(int position) {
     private void syncQueue() {
         if(!WallabagConnection.isNetworkAvailable()) {
             Toast.makeText(this, getString(R.string.txtNetOffline), Toast.LENGTH_SHORT).show();
+            Toast.makeText(this, getString(R.string.prevLoad), Toast.LENGTH_SHORT).show();
```

Comment:
Well, it can be concatenated to form one. But if we remove the second one, it would seem as if it is a bug in the app if it always pop up during the start of the app